database/sql/V048__create_settlement_rewards.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE settlements
+  ADD COLUMN id bytea default NULL,
+  ADD COLUMN gas_usd integer default NULL,
+  ADD COLUMN gas_price integer default NULL;
+
+CREATE TABLE settlement_scores (
+  settlement_id bytea NOT NULL,
+  winning_score numeric(78,0) NOT NULL,
+  reference_score numeric(78,0) NOT NULL
```

Comment:
This isn't part of the CIP, but I do think we also need to store the block deadline somewhere. Not sure if this should be tied to `auction` or `settlement_scores`, but it should go somewhere.

See https://github.com/cowprotocol/services/pull/1166#issuecomment-1411869152

Reply:
I think this comment was not addressed before merging.

Reply:
Yes! Great point.... Should we prepare an issue to track this better?

Reply:
Addressed here: https://github.com/cowprotocol/services/issues/1108#issuecomment-1438371881